a3p-integration/proposals/a:upgrade-14/package.json

```diff
@@ -3,11 +3,7 @@
     "releaseNotes": "https://github.com/Agoric/agoric-sdk/releases/tag/agoric-upgrade-14",
     "sdkImageTag": "unreleased",
     "planName": "agoric-upgrade-14",
-    "upgradeInfo": {
-      "coreProposals": [
-        "@agoric/vats/scripts/build-wallet-factory2-upgrade.js"
-      ]
-    },
+    "upgradeInfo": {},
```

Comment:
I see that the upgrade core-eval script reference into the Go code. I agree that's better, to make an explicit dependency of the upgrade-14 process. (This `upgradeInfo` field doesn't necessarily reflect the actual proposal.)

However, that's pretty buried for someone to know what's included upgrade-14. Please make or update a README.md in this proposal dir indicating that the proposal is part of it. Then also omit the upgradeInfo key here since it's empty.